src/main/java/net/pms/dlna/DLNAMediaDatabase.java

```diff
@@ -105,8 +105,9 @@
 	 *       to 1.4.196 because 1.4.197 broke audio metadata being
 	 *       inserted/updated
 	 * - 23: Store aspect ratios as strings again
+	 * - 24: added MusicBrainzID to audio tracks
 	 */
-	private final int latestVersion = 23;
+	private final int latestVersion = 24;
```

Comment:
Do you think it would be possible to do this without bumping the database version? I am hesitant to do that since it is quite an expensive operation to happen on people's machines.

Maybe you can use the existing `MusicBrainzReleases` table and add a `AudioTrackID` column?

Reply:
Sure, I'll have a look at it ... there seems to be a method for this use case.